opencti-platform/opencti-front/src/components/ItemIcon.jsx

```diff
@@ -314,6 +314,7 @@ const iconSelector = (type, variant, fontSize, color, isReversed) => {
         <SafetyCheckOutlined style={style} fontSize={fontSize} role="img" />
       );
     case 'dashboard':
+    case 'workspace':
```

Comment:
Yes for that case, it's due to the stream part. When you go to "Activity" in Entreprise Edition, you can access a stream and the icon was missing because the stream take the name of the entity type in the back, in our case workspace to define an icon. But in other part of the platform the icon will take 'dashboard' as a reference. 
So we put them both. In the case/switch mode, the dashboard will take the same property as the one below.